cartography/intel/gcp/storage.py

```diff
@@ -82,6 +98,26 @@ def get_gcp_buckets(storage: Resource, project_id: str, common_job_parameters) -
         else:
             raise
 
+@timeit
+def get_storage_policy_entities(storage,name,project_id):
+
+    try:
+        iam_policy = storage.buckets().getIamPolicy(bucket=name).execute()
+        bindings = iam_policy.get('bindings', [])
+        entity_list, public_access = iam.transform_bindings(bindings, project_id)
```

Comment:
same here as well. Since the same code is used in multiple places, can we move that to a common place (outside of iam)?